### ISSUE TITLE
app's javascript 'stops' when another app gets focus

### ISSUE THREAD

**Issue by ajuga2** [android-js/androidjs#137]
'myapp' functions normally when it has 'focus', But when you activate another app (for example a brower). the front (index.html) and back (main.js) of myapp stops. But myapp isn't closed. When you click on myapp, index.html is reloaded

**Comment by Chhekur** [comment]
have you restricted your apps to be running in background?

**Comment by ajuga2** [comment]
No, I use story-app as a template and modify main.js. I haven't modified package.json, etc

**Comment by Chhekur** [comment]
then it might be a bug, we'll look into it.
join our slack community too, for faster responses.
[Slack Channel](https://join.slack.com/t/androidjs/shared_invite/zt-e0y0u6z6-hZdwdv1zN9AMbDKxhp7NcQ) for support and discussion

**Comment by ajuga2** [comment]
It doesn't stop immediately but after about 30 seconds. I've tried using setTimeout to send messages between front and back every second but that has no effect. It looks like the OS stops the front and back processes. When the app gets focus again, it starts with a white screen after which view.html is loaded.

**Comment by notYou263** [comment]
further is it possible to use wake_lock to keep cpu running?

**Comment by ajuga2** [comment]
It is not the phone which goes to 'sleep'.

**Comment by omarcosr** [comment]
Has anyone managed to resolve this issue?
